source/main/utils/blockSyncProgressHelpers.ts

```diff
@@ -1,8 +1,7 @@
 import moment, { Moment } from 'moment';
 
 export function isItFreshLog(applicationStartDate: Moment, line: string) {
-  const [, logDate] = line.match(
-    /\[(\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d+) UTC]/
-  );
+  const [, logDate] =
+    line.match(/\[(\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d+) UTC]/) || [];
```

Comment:
Please check also if the `logDate` is not `undefined`. In case it is the `moment.utc(undefined)` will return the current time so this function may return false positive. Maybe it would be worth to return false if the `logDate` was not matched.

Reply:
Updated, thanks!